code/modules/vending/clothesmate.dm

```diff
@@ -185,7 +185,10 @@
 		            /obj/item/clothing/shoes/singery = 1,
 		            /obj/item/clothing/shoes/singerb = 1,
 		            /obj/item/clothing/shoes/yogs/trainers = 1,
-		            /obj/item/clothing/shoes/yogs/fuzzy_slippers = 3) //yogs clothes for clothesmate end here
+		            /obj/item/clothing/shoes/yogs/fuzzy_slippers = 3,
+					/obj/item/clothing/suit/jacket/pride = 3,
+					/obj/item/clothing/suit/jacket/pride/mlm = 2,
+					/obj/item/clothing/suit/jacket/pride/lesbian = 2) //yogs clothes for clothesmate end here
```

Comment:
wtf is going on here
is your pr updated to master?
why isnt the suits in contraband on this branch but it is on master? you're not removing it or anything

Reply:
weird